### ISSUE TITLE
Sector Mechanic to increase difficulty (Can't received launches from lunchpads, maybe dinner pads too)

### ISSUE THREAD

**Issue by itcannotbe** [Anuken/Mindustry-Suggestions#2956]
### Describe the content or mechanics you are proposing.

A sector that can't receive from launch pads

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

A way to increase difficulty for a certain sector.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by joshuaptfan** [comment]
Yes, could be a map rule you can set in the editor. Would counter the resource snowball effect that makes campaign no longer challenging.

**Comment by OnyxFinger** [comment]
I would say yes
and in lore it would be a sector where the enemy has a cannon of some sort to intercept launched ressources maybe ?

**Comment by itcannotbe** [comment]
May or may not include core schematics except core and it's loadout for increased starting difficulty. LOL.

**Comment by Ymabert** [comment]
@itcannotbe it makes sense, but what characteristics must that sector have that it cannot receive?

**Comment by vinhsad** [comment]
~~enemy got a big gun that shoot down your 250 surge lunch pod~~

**Comment by genNAowl** [comment]
Haha I honestly want to delete launchpads in general and see campaign players squirm.

**Comment by OnyxFinger** [comment]
> ~enemy got a big gun that shoot down your 250 surge lunch pod~

~yes yes ahahaha~

**Comment by GamerMoment69** [comment]
so basically preventing a sector from getting lunch, dinner, and maybe breakfast too

**Comment by Dan-Gamin** [comment]
I think the harder a sector is, the less the chance of a payload arriving successfully at its destination. (for example: low 90%, medium 50%, high 40%, intense 20%, extreme 5%). 

and maybe enemy bases could that the more aerial turrets it has, the less the chance of succeeding at arrival. or maybe special turrets. It would be also nice if enemy bases also got resources from other sectors, giving a player an incentive to build anti-payload turrets themselves.

A fuel requirement could also nerf the launch pads even more.
